templates/host_template.conf.j2

```diff
@@ -0,0 +1,23 @@
+// {{ ansible_managed }}
+{% import 'object_attributes.j2' as attributes %}
+
+object Host "{{ item }}" {
+   {% if hostvars[item].i2_host_template is defined -%}
+   import "{{ hostvars[item].i2_host_template }}"
+   {% else -%}
+   import "generic-host"
+   {% endif -%}
+   address = "{{ hostvars[item].ansible_default_ipv4.address }}"
+
+   vars.os = "{{ hostvars[item].ansible_system }}"
```

Comment:
Should these two really be hardcoded or just be handled like any other variable?

Reply:
In the second case, we could get rid of this template and re-use the generic object template.